### DISCUSSION TITLE
unable to activate ambient light

### DISCUSSION THREAD

**Discussion by ttkrise** [cedricp/ddt4all#1240] [Q&A]
Hello, I changed the 7 inch screen to 8.7 but the ambient light no longer works.

**Comment by Furtif** [comment]
🤔🤔🤔🤔

**Comment by AndyIreland** [comment]
yes , because the ambient features are in the multimedia - you will have to program your new one

**Reply by ttkrise** [reply]
yes I have already modified my elm to enter the multimedia can and change the parameters but it still does not work

**Reply by AndyIreland** [reply]
how old is the car? - im wondering is the obd2 gateway locked? - did it save the new parameters ok?

**Reply by ttkrise** [reply]
it's a 2016 megane iv yes it has recorded the modifications

**Reply by AndyIreland** [reply]
that should be OK then - did you 'reset hard' after programming or re-boot the radio after programming? , not say you should do and be careful because could brick the unit, but sometimes you have to hard reset to make changes. - when you done magnifying glass to search for radionav unit what radionav unit did it find?

**Reply by kartoffelpflanze** [reply]
@AndyIreland Why do you say a hard-reset could brick the unit? As I know it, it's simply a way to reboot it without the hassle of removing its power supply.

**Reply by ttkrise** [reply]
no I had never done hard rest I just did it I send you my ddt4all config is radnav 1.4 to 2.0

**Reply by ttkrise** [reply]
![20240910_105519](https://github.com/user-attachments/assets/5babb373-2273-4c4a-9125-df502f9687a8)

![Screenshot_20240910_101915_Ecu Tweaker](https://github.com/user-attachments/assets/9667818a-6373-448c-8eb8-4ca2d6766ea8)
![Screenshot_20240910_101934_Ecu Tweaker](https://github.com/user-attachments/assets/9e251e10-b804-4627-8da0-803473351ec8)
![Screenshot_20240910_102056_Ecu Tweaker](https://github.com/user-attachments/assets/7b3fd9f6-6513-4421-b032-6263c8190bd1)
![Screenshot_20240910_102122_Ecu Tweaker](https://github.com/user-attachments/assets/ff7b17cf-6304-4cc8-8a5e-0f2d915cae63)
![Screenshot_20240910_102145_Ecu Tweaker](https://github.com/user-attachments/assets/4069790d-1627-46e2-950b-be7d6d656683)
![Screenshot_20240910_102255_Ecu Tweaker](https://github.com/user-attachments/assets/688c933b-2626-4c1f-9265-037f1326deda)
![Screenshot_20240910_111158_Ecu Tweaker](https://github.com/user-attachments/assets/c0fdde6d-102c-491d-a967-4d5630951c32)
![Screenshot_20240910_111221_Ecu Tweaker](https://github.com/user-attachments/assets/f82cbce3-be79-4acd-85bd-731f22f74246)
![Screenshot_20240910_111331_Ecu Tweaker](https://github.com/user-attachments/assets/3b115b8f-3bf6-4a0b-be3f-1653fdd0b046)

**Reply by AndyIreland** [reply]
> @AndyIreland Why do you say a hard-reset could brick the unit? As I know it, it's simply a way to reboot it without the hassle of removing its power supply.

because some people might have mis-read it and press the 'reset soft' and then be in trouble

**Reply by kartoffelpflanze** [reply]
> people might have mis-read it

That's on the user then, the button itself is safe =) This isn't the type of software to mess with if you're prone to misreading.

**Reply by AndyIreland** [reply]
i have talked with confident knowlegeable/ capable people who have still pressed the 'reset soft' button instead and immediately realised and knew they shouldnt have pressed it or have pressed it in error instead of the 'reset hard button' - they are right beside each other

**Reply by AndyIreland** [reply]
> no I had never done hard rest I just did it I send you my ddt4all config is radnav 1.4 to 2.0

after you change settings do you 'hard reset' and reboot radio and it still dont work?
![image](https://github.com/user-attachments/assets/241d0289-b951-4613-876b-f8eef0c89e09)

Did you pick radnav 1.4 to 2x manually or is this what magnifying search found?

**Reply by ttkrise** [reply]
This is what the magnifying glass found or I turned off the radio by pressing power for 5 seconds and I also did a hard reset

**Comment by AndyIreland** [comment]
@ttkrise  - sorry if this is a bit of wrong information , because I dont have any ambient lights on any of my cars , and I have never set any within DDT4ALL/ECUTWEAKER but I see in your supplied screenshots some you have a 'state' value of '0' - as far as i can tell '0'=light off (or disabled) - '1' is LED on but at its lowest brightness it can go - '255' is LED on and at the brightness it can go , maybe you could try adjusting the 'state' of the lights that are not turning on from '0' to '255' and write them values and see if that enables the ambient lights you want in your car - but as I say , I am sorry if I am feeding you false information.

**Reply by ttkrise** [reply]
yes I have already tried to put everything at 255 but it did not work, I reset the lights that I do not have in the car to 0

**Comment by AndyIreland** [comment]
@ttkrise - have you manually tried selecting Navigation ECU A-IVI v.09 seems to come up for Megane IV XFB 
That also has MEX Ambilite settings:
![image](https://github.com/user-attachments/assets/cb227440-58d4-4634-bde1-4992d38f5094)

If it a Megane IV PH2 there are even more A-IVI to choose from if you have to manually , if magnifying glass search has not found the Navigation automatically 

![image](https://github.com/user-attachments/assets/c9e7d465-e71c-4437-8137-5cddd5291c93)

- person always has to be cautious when selecting an ECU manually though , be careful

**Reply by kartoffelpflanze** [reply]
Sorry for my own possible misinformation, but isn't "A-IVI" for the Easylink, and "RadNav" for the R-Link, a.k.a. completely different things?

**Reply by AndyIreland** [reply]
yes but its been replaced with a larger unit I think , might even be a completely different unit I dont know. I dont even know if the 2016 Megane 4 is a ph1 or ph2. 

if clicked on A-IVI and the unit is not that then all (should) come up with an incompatible one chosen should be a load of red boxes with 'No Data' in them if its the wrong one when reading , it shouldnt be dangerous selecting 'Read' - its 'Write' i would be cautious of especially if picking a wrong ECU out of the list - possibly you might get a 'bad ELM - function not supported' but then again it could brick the unit

**Reply by kartoffelpflanze** [reply]
@ttkrise How about you show us the unit you replaced, and what you replaced it with 😌

**Reply by AndyIreland** [reply]
that would be a good idea ....

**Reply by ttkrise** [reply]
it's megane 4 from 2016 ph 1 in firmware 3.3.16.961 I just changed from a 7 inch to an 8.7 inch and I have no data to a-ivi
![20240910_154248](https://github.com/user-attachments/assets/ad202928-2cb2-4b4d-8548-0f984f0a4958)
![20240910_154254](https://github.com/user-attachments/assets/95295b30-f9b1-41f1-95ce-765c4a46a27a)

**Reply by AndyIreland** [reply]
ah right thanks - yep definately R-Link2 - I didnt even realise you could get the rlink2 in different sizes. I have only ever seen then them in the size you have in the second photo

**Reply by ttkrise** [reply]
but I don't think that the radnav configuration has anything to do with the ambient light I have a friend who didn't have the lights in the car he changed the door sides with gt line the car was pre-wired and he went from his original 7 inch screen to a large screen and the lights came on without configuring anything

**Reply by AndyIreland** [reply]
I would have thought its all set by the radionav , multimedia screen. and if you had the lights working before with your small screen and you change to big screen and lights do not work now , that surely would indicate that there is something not set properly in the radionav settings in DDT4ALL no? 

There is a setting for MEX in the BCM/UCH in the mmi section , but again I cannot how changing your rlink to a bigger screen would disable MEX in that section

**Reply by ttkrise** [reply]
I just tested my friend's screen on my car and the lights came back on and we tested my screen on his car and the lights don't come on anymore

**Reply by AndyIreland** [reply]
you would think then that there are a few more adjustments you have to make on your screen then with DDT4ALL/ECUTWEAKER then

**Reply by ttkrise** [reply]
Can the screen come from a range without ambient light, so it doesn't work?

**Reply by AndyIreland** [reply]
i cannot answer that sorry, i dont know much about the rlink2 - but what about versions of software, did you check that too - has your friends one got the same software version on his as yours? 

how about wiring connectors at the back of the unit? - is your friends wiring at the back of his exactly the same as the wiring at the back as your car? - 

sorry I cannot be of more help , am sure someone on here more knowledgeable than I am about the rlink2 , how they work, about your issue.

**Reply by ttkrise** [reply]
my friend and in the latest version of the R-Link2 but the wiring seems identical but I think that the R-Link2 really has no importance it is really when I change the screen that the lights no longer work it does not matter you have already helped me a lot

**Reply by AndyIreland** [reply]
thank you - i thought it might have given you some pointers at least - just shame I didnt manage to have full solution for you . 
I could be wrong but I think you have 2 components there same like the rlink1 - so you have the radio tucked up in the dash somewhere and then you have the touchscreen which is a MFD (multifunction display) so when you are swapping the touchscreen over to your friends I think you are just swapping over the MFD only - I could be wrong , I only have r-link1 unit in mine. 
I am surprised that when you put your display on the other car some anti-theft or something didnt kick in an not allow it to be connected to the other car? - i thought they were protected against theft those units?

**Reply by ttkrise** [reply]
Normally there are only the R-Link2 behind the dashboard that lock when changing cars unless you go into developer mode and when you press virgin mod the screen does not lock

**Reply by loriclbv** [reply]
Hi, i just buy some GT part for my megane 4.

I try too turn them on but impossible they do the same thing as @ttkrise , did you find the solution please ?

Thanks a lot

**Comment by artur2nd-max** [comment]
> Hello, I changed the 7 inch screen to 8.7 but the ambient light no longer works.

Hi @ttkrise .
I have the same problem with my Ambient Light.
I changed the Screen from 7" to 8.7" and my Ambient Light dont work anymore.

Did you found a solution?

Greets

**Reply by bz-gt** [reply]
You need to configure Central Panel appropriately. To be able to do that, you need to connect to CAN multimedia (pins 12 and 13 in OBD Socket)

**Reply by loriclbv** [reply]
> You need to configure Central Panel appropriately. To be able to do that, you need to connect to CAN multimedia (pins 12 and 13 in OBD Socket)

Hello, i just buyed some GT parts with the ambiant lights in the doors. i pluged them with the cable who came out from the door and did the settings in ddt4all but when i turned them on the setting go back to off without never get any light from the door (idk if it's clear i do my best)

do you have any idea ? 
it's a 2016 model with the business finition 

can you explain the 12 and 13 CAN thing with more detail please ? 

Thanks a lot

**Reply by bz-gt** [reply]
> > You need to configure Central Panel appropriately. To be able to do that, you need to connect to CAN multimedia (pins 12 and 13 in OBD Socket)
> 
> Hello, i just buyed some GT parts with the ambiant lights in the doors. i pluged them with the cable who came out from the door and did the settings in ddt4all but when i turned them on the setting go back to off without never get any light from the door (idk if it's clear i do my best)
> 
> do you have any idea ? it's a 2016 model with the business finition
> 
> can you explain the 12 and 13 CAN thing with more detail please ?
> 
> Thanks a lot

It is because you are not connected to the Central Panel. Regarding Multimedia CAN, look at this repo readme
https://github.com/[KarelSvo/ddt4all-5.6.0](https://github.com/KarelSvo/ddt4all-5.6.0)
